src/CSharp/CodeCracker/Refactoring/XmlDocumentationAnalyzer.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+
+namespace CodeCracker.CSharp.Refactoring
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class XmlDocumentationAnalyzer : DiagnosticAnalyzer
+    {
+        internal const string Title = "You have missing/unexistent parameters in Xml Docs";
+        internal const string MessageFormat = "You have missing/unexistent parameters in Xml Docs";
+        internal const string Category = SupportedCategories.Refactoring;
+
+        internal static DiagnosticDescriptor XmlDocumentationRule = new DiagnosticDescriptor(
+            DiagnosticId.XmlDocumentation.ToDiagnosticId(),
+            Title,
+            MessageFormat,
+            SupportedCategories.Refactoring,
+            DiagnosticSeverity.Warning,
+            isEnabledByDefault: true,
+            helpLinkUri: HelpLink.ForDiagnostic(DiagnosticId.XmlDocumentation));
```

Comment:
Category should be maintenability. Needs to change the `DiagnosticSeverity` and file locations.